### DISCUSSION TITLE
[Apple Music] Support adding a track to Favorites

### DISCUSSION THREAD

**Discussion by sheriferson** [rekkyrosso/ampcast#24] [Ideas]
Mostly described in title. I like adding tracks to my Favorites (usually represented in the Apple Music app with the star icon, and represented in an app like Marvis Pro with the heart icon). Would be great to be able to do that (and remove the track from favorites too) from ampcast.

**Comment by rekkyrosso** [comment]
I don't think the Apple Music API has an endpoint for this. Is it possible in Marvis Pro?

**Reply by sheriferson** [reply]
It _is_ possible in the iOS app. I'm not sure how it's done, but liking/heart-ing a track that Marvis Pro is playing through Apple Music adds the track to Apple Music favorites. Maybe it's an iOS API and not an Apple Music service API?

**Comment by rekkyrosso** [comment]
I looked into this and I don't think it's currently possible using the Web API. I'm happy to be proven wrong though.
